helix-term/src/ui/completion.rs

```diff
@@ -230,7 +231,7 @@ impl Completion {
 
             match event {
                 PromptEvent::Abort => {}
-                PromptEvent::Update => {
+                PromptEvent::Update => if preview_completion_insert {
```

Comment:
this will fail exhaustivenss checks. Have you tested this code?

Reply:
Ah I got  tripped up by the if not being in braces that looked like a match guard 9which it probably should be)

Reply:
An empty arm was need anyway, fixed it now. 👍